### PR TITLE
[Vue][SIMS #1198] - Changes to all student views to use student-page-container

### PR DESCRIPTION
**Pull request by dheepak-aot**
Changes to all student views to use student-page-container

### CODE REVIEW

File: sources/packages/web/src/views/student/NoticeOfAssessment.vue [hunk 1]
```diff
@@ -11,27 +11,29 @@
...
-  <full-page-container>
-    <notice-of-assessment-form-view :assessmentId="assessmentId" />
-    <v-row class="justify-center mt-4">
-      <v-btn color="primary" @click="confirmAssessment()">
-        Confirmation of assessment
-      </v-btn>
-    </v-row>
-  </full-page-container>
+    </template>
+    <template #content>
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/student/NoticeOfAssessment.vue:15]: Is it not the default one? If yes we can omit it.

File: sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue [hunk 2]
```diff
@@ -19,16 +19,20 @@
...
-        formName="studentapplication"
-        :data="initialData"
-        @loaded="formLoaded"
-        @customEvent="customEventCallback"
-      ></formio>
-    </v-container>
-  </v-sheet>
+  <student-page-container>
+    <template #content>
+      <v-sheet elevation="1" class="mx-auto">
```
Comment (ann-aot) [inline sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue:24]: we can use v-card instead of v-sheet

Comment (dheepak-aot) [reply sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue:24]: I didn't change any existing code to keep it simple.. checked the UI if it looked like before 😊

File: sources/packages/web/src/views/student/StudentApplicationDetails.vue [hunk 3]
```diff
@@ -1,67 +1,69 @@
...
 import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
 import CancelApplication from "@/components/students/modals/CancelApplicationModal.vue";
-import RestrictionBanner from "@/components/students/RestrictionBanner.vue";
 import { ApplicationService } from "@/services/ApplicationService";
 import "@/assets/css/student.scss";
 import { useFormatters, ModalDialog, useToastMessage } from "@/composables";
 import { GetApplicationDataDto, ApplicationStatus } from "@/types";
 import ApplicationDetails from "@/components/students/ApplicationDetails.vue";
 import ConfirmEditApplication from "@/components/students/modals/ConfirmEditApplication.vue";
+import HeaderNavigator from "@/components/generic/HeaderNavigator.vue";
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/student/StudentApplicationDetails.vue:66]: This is part of the main.ts, we do not need to declare it.

File: sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue [hunk 4]
```diff
@@ -44,9 +48,10 @@ import { FormIOCustomEvent, FormIOCustomEventTypes } from "@/types";
 import { ApplicationService } from "@/services/ApplicationService";
 import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
 import { ProgramYearService } from "@/services/ProgramYearService";
+import StudentPageContainer from "@/components/layouts/student/StudentPageContainer.vue";
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue:51]: This is part of the main.ts, we do not need to declare it.

**Review by andrewsignori-aot — COMMENTED** [review]
Thanks for doing this. Please take a look at the comments.

File: sources/packages/web/src/views/student/NoticeOfAssessment.vue [hunk 5]
```diff
@@ -11,27 +11,29 @@
...
+    </template>
+  </student-page-container>
 </template>
 
 <script lang="ts">
 import NoticeOfAssessmentFormView from "@/components/common/NoticeOfAssessmentFormView.vue";
 import { useToastMessage } from "@/composables";
 import { StudentAssessmentsService } from "@/services/StudentAssessmentsService";
 import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
+import StudentPageContainer from "@/components/layouts/student/StudentPageContainer.vue";
```
Comment (ann-aot) [inline sources/packages/web/src/views/student/NoticeOfAssessment.vue:31]: u have already  imported in main

File: sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue (hunk 4 above)
Comment (ann-aot) [inline sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue]: already imported in main

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by henryoforeh-aot — APPROVED** [review]

**Review by ann-aot — APPROVED** [review]
Thanks for doing the changes 👍

**Review by guru-aot — APPROVED** [review]
LGTM

**Pull request merged**